packages/framework/build.xml

```diff
@@ -762,6 +762,7 @@
             <arg path="${path.tests}"/>
 
             <arg value="--level=${phpstan.level}"/>
+            <arg value="--memory-limit=4000M"/>
```

Comment:
Interesting number - why have you chose this one exact?

Reply:
In some example. Should I change it?

Reply:
Well we have set in php.ini 512MB, this is almost eight time increase, I found it a lot, also when we suggest for example in Docker Installation guide that 4GB of memory is sufficient for whole SSFW in Docker :)

Reply:
:+1: 061b389dc4402e02bb25c1909e70bcc885ad9c77